R-packages/covidcast/R/plot.R

```diff
@@ -411,16 +414,22 @@ plot_line = function(x, range = NULL, col = 1:6, line_type = rep(1:6, each = len
     ggplot2::theme(legend.position = "bottom",
                    legend.title = ggplot2::element_blank())
 
-  # Create line and lim layers
-  line_layer = ggplot2::geom_line(ggplot2::aes(x = time_value, y = value,
+  # Create lim, line, and ribbon layers
+  lim_layer = ggplot2::lims(y = range)
```

Comment:
Do you really want `lims`? Values outside the range will be deleted entirely. That means that if, say, the user sets a range of [0, 10], and one value happens to be 11, you won't see the graph spike off the top and come back down -- it'll just connect the two surrounding points with a straight line.

You probably want to use [coord_cartesian](https://ggplot2.tidyverse.org/reference/coord_cartesian.html) to set the ylim.

(And if this annoys you, this was also Howard's biggest peeve about ggplot2, so you're not alone)

Reply:
Good catch!  That's pretty annoying alright.  However, not my biggest peeve: there are still plenty of other contenders for the top spot. 😉